src/binder/bind/bind_call.cpp

```diff
@@ -0,0 +1,29 @@
+#include "binder/binder.h"
+#include "binder/call/bound_call.h"
+#include "common/string_utils.h"
+#include "parser/call/call.h"
+
+namespace kuzu {
+namespace binder {
+
+std::unique_ptr<BoundStatement> Binder::bindCallClause(const parser::Statement& statement) {
+    auto callStatement = reinterpret_cast<const parser::Call&>(statement);
```

Comment:
Notice that you are copying since you are not using `auto&`

Reply:
added `auto&`